### PR TITLE
Add support for sonar.search.host and sonar.search.port properties

### PR DESCRIPTION
**Pull request by nwheeler**

### CODE REVIEW

File: manifests/init.pp [hunk 1]
```diff
@@ -49,6 +49,8 @@
   $profile          = false,
   $web_java_opts    = undef,
   $search_java_opts = undef,
+  $search_host = undef,
```
Comment (racodond) [inline manifests/init.pp]: @nwheeler Could you please set default values and align the '=' sign?

File: templates/sonar.properties.erb [hunk 2]
```diff
@@ -122,6 +122,18 @@ sonar.web.https.keystoreType=<%= @https['keystoretype'] %>
 sonar.search.javaOpts=<%= @search_java_opts %>
 <% end -%>
 
+<% if @search_port -%>
```
Comment (racodond) [inline templates/sonar.properties.erb]: @nwheeler Could you please remove  the else statements as default values should be set in `init.pp`?

**Comment by racodond** [comment]
@nwheeler Thanks for your contribution! I'll let you take into account my suggestions and I'll merge.

**Comment by nwheeler** [comment]
Okay no problem. Let me know if there's anything else I missed. Thanks!

**Comment by racodond** [comment]
Looks good. Thanks!

**Pull request merged**